xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
> If we put that line into `.gemspec`, you'll have Rouge installed on each Travis job.

You wouldn't, both Circle CI and Travis CI pre-install `xcpretty` in their OS X images.

I've removed it from the gem spec in 336e394 since you feel strongly about it not being there. But this means that RubyGems will not lock the version of Rouge and any breaking changes could cause xcpretty to break for users either now or in the distant future.

If the user has Rouge < 1.8 it may not work, or >= 2.0. 

---

> We should be using Homebrew and treat xcpretty like a CLI tool rather than a library.

Isn't that an even bigger dependency? One that requires you to be an administrator of your computer. Many users may not be able install `xcpretty` if they are in corporate or education environments where they do not have administrator privileges.

This conversation is off-topic and doesn't belong on this PR...